examples/tests/stress.py.data/stress.yaml

```diff
@@ -0,0 +1,9 @@
+setup:
+    duration:
+        stress_lenght: 60
+    workers:
+        threads: null
+    memory:
+        memory_per_thread: null
+    files:
+        file_size_per_thread: null
```

Comment:
Maybe, since this is both a port and an example, we could provide different variants (2 or 3) with different levels of aggressiveness and test length.